legal-api/src/legal_api/models/filing.py

```diff
@@ -740,9 +740,9 @@ def set_processed(self, business_type):
             self.effective_date = self.payment_completion_date
 
     def effective_date_can_be_before_payment_completion_date(self, business_type):
-        """For AR or COD filings on CP or BEN then the effective date can be before the payment date."""
+        """For AR or COD filings then the effective date can be before the payment date."""
         return self.filing_type in (Filing.FILINGS['annualReport'].get('name'),
-                                    Filing.FILINGS['changeOfDirectors'].get('name')) and business_type in {'CP', 'BEN'}
+                                    Filing.FILINGS['changeOfDirectors'].get('name'))
```

Comment:
Are there any other business types that might get here and that want to handle differently?

Reply:
Never mind - answered by Vysakh. Resolving.